nuxeo-ai-model/src/main/java/org/nuxeo/ai/model/export/DatasetExportServiceImpl.java

```diff
@@ -293,4 +321,41 @@ protected String notNullNxql(String nxql, List<Map<String, String>> featuresWith
         }
         return buffy.toString();
     }
+
+    /**
+     * For the given property, find out if it exists and determine if its text or content
+     */
+    protected IOParam getPropertyWithType(String prop) {
```

Comment:
Why did you move these methods?

Reply:
they were in another module but used in this service. I needed these helpers to be able to work with `IOParam` and to avoid cycling deps I moved them where they belong

Reply:
It seems like you are more likely to be creating a cyclic dependency.  The dependencies go : pipe -> core -> model -> package.  (Model was not included for 9.10 but after all, we didn't do a release for 9.10)

Reply:
If you are saying its only used in 1 place and you are not duplicating code then I guess its ok to move